{"name": "显示满足条件的课程信息（输入存在的课程介绍，课程名称，课程学科，课程价格，适用人群）", "status": "failed", "statusDetails": {"message": "AssertionError: assert ('\\n'\\n '### Error updating database.  Cause: '\\n 'com.mysql.cj.jdbc.exceptions.MysqlDataTruncation: Data truncation: Data too '\\n \"long for column 'subject' at row 1\\n\"\\n '### The error may exist in URL '\\n '[jar:file:/root/kdtx/huike-admin.jar!/BOOT-INF/lib/huike-clues-3.4.0.jar!/mapper/clues/TbCourseMapper.xml]\\n'\\n '### The error may involve '\\n 'com.huike.clues.mapper.TbCourseMapper.insertTbCourse-Inline\\n'\\n '### The error occurred while setting parameters\\n'\\n '### SQL: insert into tb_course          ( code,             '\\n 'name,             subject,             price,             '\\n 'applicable_person,             info,             create_time )           '\\n 'values ( ?,             ?,             ?,             ?,             '\\n '?,             ?,             ? )\\n'\\n '### Cause: com.mysql.cj.jdbc.exceptions.MysqlDataTruncation: Data '\\n \"truncation: Data too long for column 'subject' at row 1\\n\"\\n \"; Data truncation: Data too long for column 'subject' at row 1; nested \"\\n 'exception is com.mysql.cj.jdbc.exceptions.MysqlDataTruncation: Data '\\n \"truncation: Data too long for column 'subject' at row 1\") == '操作失败'\n  - 操作失败\n  + \n  + ### Error updating database.  Cause: com.mysql.cj.jdbc.exceptions.MysqlDataTruncation: Data truncation: Data too long for column 'subject' at row 1\n  + ### The error may exist in URL [jar:file:/root/kdtx/huike-admin.jar!/BOOT-INF/lib/huike-clues-3.4.0.jar!/mapper/clues/TbCourseMapper.xml]\n  + ### The error may involve com.huike.clues.mapper.TbCourseMapper.insertTbCourse-Inline\n  + ### The error occurred while setting parameters\n  + ### SQL: insert into tb_course          ( code,             name,             subject,             price,             applicable_person,             info,             create_time )           values ( ?,             ?,             ?,             ?,             ?,             ?,             ? )\n  + ### Cause: com.mysql.cj.jdbc.exceptions.MysqlDataTruncation: Data truncation: Data too long for column 'subject' at row 1\n  + ; Data truncation: Data too long for column 'subject' at row 1; nested exception is com.mysql.cj.jdbc.exceptions.MysqlDataTruncation: Data truncation: Data too long for column 'subject' at row 1", "trace": "self = <test_case.add_center.test_data_add.Test_add object at 0x0000020A729E1DC0>\nlogin_fixture = 'eyJhbGciOiJIUzUxMiJ9.eyJsb2dpbl91c2VyX2tleSI6IjlkMGI5Y2IwLTBjOGMtNDI0MS1hMGQ0LTIxZGEyMGFhODUzMCJ9.Hir_4Ln_cUyM2iq-d4dM-GoLMEqT-K7RxnZ8h3m3Z5ZqlPi4XDw26XAZApoARM52OpGcoj_8B3KMgKpUnOo5JQ'\nname = '软件开发课01', subject = 66, price = 999, applicablePerson = 3, info = ''\n\n    @allure.severity(severity_level=\"critical\")\n    @pytest.mark.parametrize(\"name,subject,price,applicablePerson,info\", get_data()['add2'])\n    def test_add_02(self,login_fixture,name,subject,price,applicablePerson,info):\n        headers = {\n            \"Authorization\": login_fixture\n        }\n        json={\n            \"name\": name,\n            \"subject\": subject,\n            \"price\": price,\n            \"applicablePerson\":applicablePerson,\n            \"info\": info\n        }\n        ######动态自定义allure报告，title和story的信息写在yaml文件中\n        title = get_data()['text']['title']\n        story = get_data()['text']['story']\n        allure.dynamic.title(title)\n        allure.dynamic.story(story)\n        ##########################################\n        #r = api_utils.api_search(params=params,headers=headers)    ###关键字传参\n        r = response.post_add_query(headers=headers,json=json)\n        assert r.status_code == 200\n        result=r.json()\n>       assert result['msg']== \"操作失败\"\nE       assert ('\\n'\\n '### Error updating database.  Cause: '\\n 'com.mysql.cj.jdbc.exceptions.MysqlDataTruncation: Data truncation: Data too '\\n \"long for column 'subject' at row 1\\n\"\\n '### The error may exist in URL '\\n '[jar:file:/root/kdtx/huike-admin.jar!/BOOT-INF/lib/huike-clues-3.4.0.jar!/mapper/clues/TbCourseMapper.xml]\\n'\\n '### The error may involve '\\n 'com.huike.clues.mapper.TbCourseMapper.insertTbCourse-Inline\\n'\\n '### The error occurred while setting parameters\\n'\\n '### SQL: insert into tb_course          ( code,             '\\n 'name,             subject,             price,             '\\n 'applicable_person,             info,             create_time )           '\\n 'values ( ?,             ?,             ?,             ?,             '\\n '?,             ?,             ? )\\n'\\n '### Cause: com.mysql.cj.jdbc.exceptions.MysqlDataTruncation: Data '\\n \"truncation: Data too long for column 'subject' at row 1\\n\"\\n \"; Data truncation: Data too long for column 'subject' at row 1; nested \"\\n 'exception is com.mysql.cj.jdbc.exceptions.MysqlDataTruncation: Data '\\n \"truncation: Data too long for column 'subject' at row 1\") == '操作失败'\nE         - 操作失败\nE         + \nE         + ### Error updating database.  Cause: com.mysql.cj.jdbc.exceptions.MysqlDataTruncation: Data truncation: Data too long for column 'subject' at row 1\nE         + ### The error may exist in URL [jar:file:/root/kdtx/huike-admin.jar!/BOOT-INF/lib/huike-clues-3.4.0.jar!/mapper/clues/TbCourseMapper.xml]\nE         + ### The error may involve com.huike.clues.mapper.TbCourseMapper.insertTbCourse-Inline\nE         + ### The error occurred while setting parameters\nE         + ### SQL: insert into tb_course          ( code,             name,             subject,             price,             applicable_person,             info,             create_time )           values ( ?,             ?,             ?,             ?,             ?,             ?,             ? )\nE         + ### Cause: com.mysql.cj.jdbc.exceptions.MysqlDataTruncation: Data truncation: Data too long for column 'subject' at row 1\nE         + ; Data truncation: Data too long for column 'subject' at row 1; nested exception is com.mysql.cj.jdbc.exceptions.MysqlDataTruncation: Data truncation: Data too long for column 'subject' at row 1\n\ntest_data_add.py:78: AssertionError"}, "attachments": [{"name": "log", "source": "82f0171a-bec8-4700-8a1c-e3a18259d2aa-attachment.txt", "type": "text/plain"}], "parameters": [{"name": "name", "value": "'软件开发课01'"}, {"name": "subject", "value": "66"}, {"name": "price", "value": "999"}, {"name": "applicablePerson", "value": "3"}, {"name": "info", "value": "''"}], "start": 1723876466321, "stop": 1723876466529, "uuid": "affd7fbc-9d06-4e81-acc5-be191e80a566", "historyId": "b2704844d15feeaa2c5f05baf0a11a59", "testCaseId": "7a199a24282d747f5dd1cb1ccc2c4d2d", "fullName": "test_case.add_center.test_data_add.Test_add#test_add_02", "labels": [{"name": "story", "value": "查询列表story"}, {"name": "severity", "value": "critical"}, {"name": "feature", "value": "添加课程"}, {"name": "parentSuite", "value": "test_case.add_center"}, {"name": "suite", "value": "test_data_add"}, {"name": "subSuite", "value": "Test_add"}, {"name": "host", "value": "DESKTOP-SSRE6IE"}, {"name": "thread", "value": "15172-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_case.add_center.test_data_add"}]}